hvcc/generators/c2wwise/c2wwise.py

```diff
@@ -31,123 +31,79 @@ class c2wwise:
 
     @classmethod
     def compile(
-        cls,
-        c_src_dir: str,
-        out_dir: str,
-        externs: Dict,
-        patch_name: Optional[str] = None,
-        patch_meta: Optional[Dict] = None,
-        num_input_channels: int = 0,
-        num_output_channels: int = 0,
-        copyright: Optional[str] = None,
-        verbose: Optional[bool] = False
+            cls,
```

Comment:
maybe silly, but why the additional indentation? :)

Reply:
Auto formatting :) PyCharm does it according to PEP 8.

Reply:
PyCharm does it according to whatever they want. The current code passes all PEP8 checks already :)

I'd be careful with full "auto-formatted" code though, it can also break style guides.

The only auto-formatting I typically do is removal of extraneous white space and addition of ending newlines.